target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -49,6 +49,12 @@ ramips_board_detect() {
 	*"ALL5003")
 		name="all5003"
 		;;
+	*"Apexera NetBot")
```

Comment:
drop the additions to this file and use the compatibles from the dts files as userspace boardname (`apexera,netbot` and `apexera,wibot`)